pkg/kubelet/kuberuntime/kuberuntime_manager.go

```diff
@@ -883,12 +884,18 @@ func (m *kubeGenericRuntimeManager) SyncPod(pod *v1.Pod, podStatus *kubecontaine
 		}
 
 		metrics.StartedContainersTotal.WithLabelValues(metricLabel).Inc()
+		if goruntime.GOOS == "windows" && sc.HasWindowsHostProcessRequest(pod, spec.container) {
```

Comment:
does this metric need to be gated behind the hostprocess feature?

Reply:
I'm not sure and was hoping to ask @ehashman this.
The metric will never be logged if the feature gate is disabled so I don't think we need to gate this call but I could be wrong.

Reply:
Also I'm not sure if the GOOS check should be left in there for the same reasons.
I didn't want to pollute metrics on Linux nodes but I'm not sure if this is necessary since the metric will never get logged on Linux nodes.